.pre-commit-config.yaml

```diff
@@ -1,21 +1,21 @@
+default_language_version:
+  node: system
+  python: python3.9
```

Comment:
Would like to ask why the movement to 3.9.

Reply:
Good spot! I had some issues running pre-commit (probably because I don't have 3.8 installed locally) I updated it to 3.11. Thanks

Reply:
Ah I see. Thanks! So we also have to move to the latest Python version.

Reply:
> Ah I see. Thanks! So we also have to move to the latest Python version.

I suppose thats OK. I really just follow our Wagtail Kit lead there 😄